src/clr-angular/forms/datepicker/_datepicker.clarity.scss

```diff
@@ -179,6 +179,7 @@
     .yearpicker-trigger {
         //Dimensions
         min-width: 2rem;
+        max-width: 2.5rem;
```

Comment:
Should the width always be 2rem?

Reply:
some locales require more space to display the month value. but we dont want it to extend more than 2.5rem.